### ISSUE TITLE
Migrate away from Less

### ISSUE THREAD

**Issue by vanillajonathan** [nextapps-de/winbox#176]
Migrate away from Less stylesheets to Sass or plain old CSS.

Less have been on the decline and other frameworks such as Bootstrap have switched from Less to Sass.

**Comment by SamMousa** [comment]
What about moving to pure CSS? Simplifies the build and this library is so small that it should suffice.

**Comment by vanillajonathan** [comment]
Yeah, I think it would be a good idea. Modern CSS nowadays support nesting too.

**Comment by SamMousa** [comment]
I'm working on a PR that moves to esbuild. It'll move us to vanilla css too

**Comment by SamMousa** [comment]
#187
